wallet-new/src/Cardano/Wallet/Kernel.hs

```diff
@@ -64,18 +68,25 @@ data DatabaseMode
 -- | A configuration type for specifying where to load the databases from.
 data DatabasePaths
     = DatabasePaths
-    { dbPathAcidState :: Maybe FilePath
-    -- ^ The path for the @acid-state@ database. If 'Nothing' is passed, then
-    -- this uses the default value (see 'openLocalState').
-    , dbPathMetadata  :: Maybe FilePath
+    { dbPathAcidState :: FilePath
+    -- ^ The path for the @acid-state@ database.
+    , dbPathMetadata  :: FilePath
     -- ^ This path is used for the SQLite database that contains the transaction
-    -- metadata. If 'Nothing' is provided, then this uses the path
-    -- @./wallet-db-sqlite.sqlite3@
+    -- metadata.
     } deriving (Eq, Show)
 
 -- | Use the default paths on disk. See 'DatabasePaths' for more details.
 useDefaultPaths :: DatabaseMode
-useDefaultPaths = UseFilePath (DatabasePaths Nothing Nothing)
+useDefaultPaths = UseFilePath
+    $ DatabasePaths
+        (defaultAcidStatePath (Proxy @DB))
+        defaultSqlitePath
+
+defaultAcidStatePath :: Typeable st => Proxy st -> FilePath
+defaultAcidStatePath p = "state" </> show (typeRep p)
```

Comment:
Interesting default name -- why not use `wallet-db-acid` for consistency?

Reply:
That's what `acid-state`'s `openLocalState` uses. I'd be happy to switch the default to that; seems more friendly.